src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -620,6 +621,168 @@ JSON example:
+
+| **Key name**         | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                                                                                                                                                                   |
+|----------------------|-----------------------|----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| IntendedFor          | REQUIRED              | [string][] or [array][] of [strings][] | One or more filenames with paths relative to the subject subfolder, with forward slashes, referring to ASL time series for which the m0scan is intended.AcquisitionVoxelSize                                                                                                                                                                                                      |
+| AcquisitionVoxelSize | RECOMMENDED           | [array][] of [numbers][]               | An array of numbers with a length of 3, in millimeters. This parameter denotes the original acquisition voxel size, excluding any inter-slice gaps and before any interpolation or resampling within reconstruction or image processing. Any point spread function effects, for example due to T2-blurring, that would decrease the effective resolution are not considered here. |
+
+The following table recapitulates the ASL field dependencies. If Source field (column 1) contains the Value specified in column 2, then the Requirements in column 4 are imposed on the Dependent fields in column 3.
+
+| **Source field**         | **Value**    | **Dependent field**  | **Requirements**                             |
+|--------------------------|--------------|----------------------|----------------------------------------------|
+| MRAcquisitionType        | 2D / 3D      | SliceTiming          | \[X\] / \[\]                                 |
+| LookLocker               | true         | FlipAngle            | \[X\]                                        |
+| ArterialSpinLabelingType | PASL         | BolusCutOffFlag      | \[X\]                                        |
+| BolusCutOffFlag          | true / false | BolusCutOffDelayTime | \[X\] / \[\]                                 |
+| BolusCutOffFlag          | true / false | BolusCutOffTechnique | \[X\] / \[\]                                 |
+| M0Type                   | Separate     | */perf/              | contains *_m0scan.nii[.gz] and *_m0scan.json |
```

Comment:
@sappelhoff Hi Stefan
I am using *_m0scan and likewise namings in the dependency table. However, I am wondering if this could screw up the table for read the docs (like it did for the [X])? I can not check this, as checks are failing now on these hyperlinks. Do you know if I should add \ here as well?

Reply:
Yes, all Special markdown characters have to be escaped using a backslash `\`,  or by putting them into an inline code block using Backtricks: `

Special markdown characters are * _ |  and so on. I usually prefer inline code blocks, but I recommend to stay consistent with how it is done elsewhere in the specification

Reply:
checks are working now so you should be able to verify @patsycle